README.md

```diff
@@ -109,8 +111,12 @@ $ docker build -t registry.ng.bluemix.net/<ibmcloud_container_registry_namespace
 $ docker push registry.ng.bluemix.net/<ibmcloud_container_registry_namespace>/watsontesseract:1
 ```
 
-* Update the `namespace` in `watson-lang-trans.yml` with `<ibmcloud_container_registry_namespace>`
- 
+* Update [`watson-lang-trans.yml`](server/watson-lang-trans.yml) with your specific values:
```

Comment:
i would think this should happen before the docker build and push... but i'll leave that up to @RiyaMRoy04 and @sanjeevghimire to confirm.